samples/Directory.Packages.props

```diff
@@ -9,7 +9,7 @@
 		This is not an issue when libraries are referenced through nuget packages. See https://github.com/unoplatform/uno/issues/446 for more details.
 		-->
 		<PackageVersion Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.2.11"/>
-		<PackageVersion Include="Microsoft.UI.Xaml" Version="2.6.0" />
+		<PackageVersion Include="Microsoft.UI.Xaml" Version="2.8.1" />
```

Comment:
is this necessary?

Reply:
I'll try to drop back to something between 2.6.0 and 2.8.1 - there's an error with 2.6.0